test/config/akamai-purger.json

```diff
@@ -24,5 +24,10 @@
   "syslog": {
     "stdoutlevel": 6,
     "sysloglevel": 6
+  },
+  "beeline": {
+      "mute": true,
+      "dataset": "Test",
+      "servicename": "AkamaiPurger"
```

Comment:
nit: I'd kind of prefer servicename to be related in some formal way to some other string - for instance the binary name?

Reply:
Yeah that's a good idea. The closest thing Go has to this is `os.Executable()`, which is the full absolute path to the executable. Would you be happy with `path.Base(os.Executable())`?

Reply:
Sure! We do something similar in VersionString in cmd/shell.go:

```
func VersionString() string {
	name := path.Base(os.Args[0])
	return fmt.Sprintf("Versions: %s=(%s %s) Golang=(%s) BuildHost=(%s)", name, core.GetBuildID(), core.GetBuildTime(), runtime.Version(), core.GetBuildHost())
}
```

os.Executable() seems a little nicer than os.Args[0], so let's update both places.

Reply:
Ooh unfortunately `os.Executable()` can return an error, so it's not a great fit for `VersionString()`. But it works well here; lemme know what you think.